cypress/integration/j5.js

```diff
@@ -0,0 +1,30 @@
+import T3Page from '../obj/tasksPage';
+import T4Page from '../obj/clientsPage';
+import LoginPage from '../obj/loginPage';
+import Common from '../obj/common';
+
+const t3page = new T3Page();
+const t4page = new T4Page();
+const loginPage = new LoginPage();
+const common = new Common();
+
+describe('Sourcebooks login', function() {
+
+    beforeEach(function () {
+        cy.loginAs("Admin");
+        cy.visit('/');
+    })
+
+    it('Admin creates new project', function(){
+
+        /* THIS T4 TASK IS NOT FINISHED */
```

Comment:
usually you should not put unfinished functionality in pull request, because it is expected that develop branch is fully working and don't spew errors, but for this lecture it;s okay